css/index.css

```diff
@@ -28,119 +28,209 @@ article, aside, details, figcaption, figure,
 footer, header, hgroup, menu, nav, section {
 	display: block;
 }
+
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    font-family: 'Roboto', sans-serif;
+}
+
 body {
+    box-sizing: content-box;
 	line-height: 1;
+    max-width: 800px;
+    padding: 5% 20% 0 20%;
 }
-ol, ul {
-	list-style: none;
+
+a {
+    text-decoration: none;
+    color: black;
+    text-align: center;
+    margin-top: 10px;
 }
-blockquote, q {
-	quotes: none;
+
+nav {
+    max-width: 600px;
+    font-size: 14px;  
+    padding-right: 50px;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
-	content: '';
-	content: none;
+
+nav a:hover {
+    color: lightcoral;
 }
-table {
-	border-collapse: collapse;
-	border-spacing: 0;
+
+p {
+    line-height: 1.4;
+    margin: 15px 0;
 }
 
-* {
-    box-sizing: border-box;
+/* 
+==============================
+flexbox
+==============================
+*/
+
+.flexbetween {
+    width: 100%;
+    display: flex;
+    flex-flow: row wrap; 
+    justify-content: space-between;
+    align-items: baseline;
+    align-content: center;
 }
 
-html, body {
-    height: 100%;
-    font-family: 'Roboto', sans-serif;
+/* 
+===============================
+button
+===============================
+*/
+
+button {
+    border-radius: 10px;
+    padding: 10px 25px;
+    font-size: 16px;
+    background: white;
+    margin: 20px 0;
+}
+
+button:hover {
+    background: black;
+    color: white;
 }
 
+/* 
+===============================
+headers
+===============================
+*/
+
 h1, h2, h3, h4, h5 {
     font-size: 18px;
-    margin-bottom: 15px;
     font-family: 'Rubik', sans-serif;
 }
 
-p {
-    line-height: 1.4;
+h1 {
+    margin-bottom: 15px;
+    text-align: left;
 }
 
-.container {
-    width: 800px;
-    margin: 0 auto;
+/* 
+==============================
+header
+==============================
+*/
+
+header {
+    margin-top: 20px;
+}
+header img {
+    max-width: 100%;
+}
+header section {
+    margin-bottom: 30px;
 }
 
-.top-content {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
-    margin-bottom: 20px;
+/* 
+==============================
+containers
+==============================
+*/
+
+.row1 {
     border-bottom: 1px dashed black;
+    padding: 20px 0;
 }
 
-.top-content .text-container {
-    width: 48%;
-    padding: 0 1%;
-    padding-bottom: 20px;  
+.lastrow {
+    border-bottom: none;
 }
-
-.middle-content {
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+.col1-2 {
+    max-width: 370px;
 }
 
-.middle-content h2 {
-    padding: 0 2%;
-    margin-bottom: 0;
+.col1-3 {
+    max-width: 246px;
 }
 
-.middle-content .boxes {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
+.textbutton {
+    margin: 10px 0 0 0;
 }
 
-.middle-content .boxes .box {
-    width: 12.5%;
+/* 
+==============================
+boxes
+==============================
+*/
+
```

Comment:
Great job using the parent/child relationships and only overriding the color on the individual box.